includes/core-testing-note-aspnet.md

```diff
@@ -0,0 +1,19 @@
+>[!IMPORTANT]
+>If you're testing an ASP.NET application or library, and you are targeting a framework _other_ than `netcoreapp3.0`, you must add an additional package reference: `Microsoft.AspNetCore.App`.
```

Comment:
I think this sentence needs some clarification. `netcoreapp3.0` is the ONLY valid TFM as of ASP.NET Core 3.0. ASP.NET Core 2.2 and earlier versions allow you to target either .NET Framework or .NET Core.

```suggestion
>If you're testing an ASP.NET Core application, and you're targeting a framework _other_ than `netcoreapp3.0`, you must add an additional package reference: `Microsoft.AspNetCore.App`.
```